tensorflow/core/ops/mkl_array_ops.cc

```diff
@@ -102,7 +102,7 @@ REGISTER_OP("_MklDequantize")
     .Input("input: T")
     .Input("min_range: float")
     .Input("max_range: float")
-    .Output("output: float")
+    .Output("output: dtype")
```

Comment:
Can you add a default of float?  Otherwise there will be a compatibility break.

Reply:
@cantonios I think it is already set to a default of float [here](https://github.com/tensorflow/tensorflow/pull/58020/files#diff-0e82ae77cc34257ccaf4ecca3a9514132fc211358058f09973cc082358f1b43cL110).

Reply:
Ah, you're right, sorry.